infrastructure/firecracker/config.go

```diff
@@ -196,6 +197,44 @@ func generateNetworkConfig(vm *models.MicroVM) (string, error) {
 	return base64.StdEncoding.EncodeToString(nd), nil
 }
 
+func configureStaticEthernet(iface *models.NetworkInterface, eth *cloudinit.Ethernet) error {
+	eth.Addresses = []string{string(iface.StaticAddress.Address)}
+
+	if iface.StaticAddress.Gateway != nil {
+		isIPv4, err := iface.StaticAddress.Gateway.IsIPv4()
+		if err != nil {
+			return fmt.Errorf("parsing gateway address: %w", err)
+		}
+
+		ipAddr, err := iface.StaticAddress.Gateway.IP()
+		if err != nil {
+			return fmt.Errorf("parsing gateway address: %w", err)
+		}
+
+		if isIPv4 {
+			eth.GatewayIPv4 = &ipAddr
+		} else {
+			eth.GatewayIPv6 = &ipAddr
+		}
+	}
+
+	if len(iface.StaticAddress.Nameservers) > 0 {
+		eth.Nameservers = &cloudinit.Nameservers{
+			Addresses: []string{},
+		}
+
+		for nsIndex := range iface.StaticAddress.Nameservers {
+			ns := iface.StaticAddress.Nameservers[nsIndex]
+			eth.Nameservers.Addresses = append(eth.Nameservers.Addresses, ns)
+		}
+	}
+
+	eth.DHCP4 = ptr.Bool(false)
+	eth.DHCP6 = ptr.Bool(false)
```

Comment:
```suggestion
	eth.DHCP4 = firecracker.Bool(false)
	eth.DHCP6 = firecracker.Bool(false)
```

to match the ones above? or the reverse, whatever we like for consistency